openedx/frontend-app-gradebook
# feat: add compatibility with transifex

Now that the frontend-app-gradebook resource was added to Transifex, we can
start pulling strings fromt there. For now, the project contains very few
translated string.

Strings were pulled by running: make pull_translations

This is for https://github.com/openedx/build-test-release-wg/issues/107

**TL;DR -**

Add compatibility with transifex.

**What changed?**

- Updated translation strings from Transifex
- Updated git-ignored files 

**Developer Checklist**

**Testing Instructions**

- Setup the site language to French in the user account
- Verify that at least some of the strings are translated when accessing the gradebook.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
CC: @sarina 

This PR will address https://github.com/openedx/build-test-release-wg/issues/107
